docs/reference/modules/discovery/no-master-block.asciidoc

```diff
@@ -0,0 +1,22 @@
+[[no-master-block]]
+=== No master block
```

Comment:
This particular page might need a better title, since if you land on it out of context it isn't very clear.  I'll review more tomorrow and see if I can suggest something.